.github/actions/install_deps/action.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Install dependencies'
```

Comment:
seems like this could just be a script in `ci/`. What do we gain by modeling it as a GH action? caching? improved display in the runner?

Reply:
Nah, workflows are too high level, this is more for reusing steps.

Reply:
@justinmk nothing really.